### ISSUE TITLE
Corepack needs help output

### ISSUE THREAD

**Issue by jasnell** [arcanis/corepack#12]
```
C:\Users\jasne>corepack

C:\Users\jasne>corepack --help

C:\Users\jasne>corepack --?

C:\Users\jasne>corepack -h

C:\Users\jasne>corepack /h
```

;-) ....

**Comment by mmarchini** [comment]
> corepack --?

😮

**Comment by arcanis** [comment]
I'll double-check, but the help should be printed on `corepack -h|--help`:

![image](https://user-images.githubusercontent.com/1037931/94601911-74a02600-0294-11eb-8ba7-5c93780e86e9.png)

it'll also be printed when running `corepack` without arguments once I upgrade Clipanion 🚀

**Comment by jasnell** [comment]
Might be that an older version pulled down from npm when I just tried it out but the output above was copied directly from my terminal :-)

**Comment by mmarchini** [comment]
maybe it's a bug on windows?

**Comment by jasnell** [comment]
Entirely possible. My ubuntu server is rebooting at the moment. Will try there a bit later

**Comment by arcanis** [comment]
I can repro when using the version from npm, will investigate 👍

**Comment by jasnell** [comment]
Yeah same issue on linux. None of the corepack commands appear to work when executed directly. The jumpers are working perfectly tho

**Comment by arcanis** [comment]
I've released the fix as `corepack@0.4.0` 🚀
